src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2021,7 +2021,7 @@ export default class Aircraft {
             }
 
             // lock ILS if at the right angle and altitude
-            if ((abs(this.altitude - glideslope_altitude) < glideslope_window)
+            if (this.altitude < (glideslope_altitude + glideslope_window)
                 && (abs(offset_angle) < degreesToRadians(10))
                 && (offset[1] < ils)
```

Comment:
Also, not something that needs to be fixed now as these large logic blocks are everywhere.  But something to keep in mind for the future.